tests/admin/test_admin_service.py

```diff
@@ -56,11 +57,12 @@ def test_handle_get_database_info_request(self):
         # Then the service responded with the expected information
         response = request_context.last_response_params
         self.assertIsInstance(response, GetDatabaseInfoResponse)
-        expected_info = {'owner': user_name}
+        expected_info = {'dbname': db_name, 'owner': user_name, 'size': None}
         self.assertEqual(response.database_info.options, expected_info)
 
         # And the service retrieved the owner name using a query with the database name as a parameter
-        mock_cursor.execute.assert_called_once_with(mock.ANY, (db_name,))
+        owner_query = "SELECT pg_catalog.pg_get_userbyid(db.datdba) FROM pg_catalog.pg_database db WHERE db.datname = '{}'".format(db_name)
```

Comment:
Why do we have to pass the owner_query? Didnt mock.ANY work?

Reply:
I changed this because the execute call was changed from passing in a query and a db_name to passing in a query with the db_name already embedded. I could change this to use `assert_called_once_with(mock.ANY)` instead, does that work?

Reply:
No lets keep it as we want to test that query used database name as a parameter. I was just thinking if there is a way where we do not have to set the query ourselves in the test.